### PR TITLE
Prepare the netcore document for alpha 2

### PR DESCRIPTION
**Pull request by bergmania**
Please note this do not have to be merged before alpha 2 is ready

### CODE REVIEW

File: UmbracoNetCoreUpdates.md [hunk 1]
```diff
@@ -63,26 +63,47 @@ Found a bug that isn't already reported? Please report it on the [GitHub tracker
...
+ * External logins are not supported yet
+ * No support for Packages currently
+
 ### Prerequisites
 
 * [.Net Core 3.1 SDK](https://dotnet.microsoft.com/download)
 * SQL connection string (MS SQL Server/Azure), unless you want to install using SQL CE (Compact Edition)
 
-### Steps to install
+### Steps to install the umbraco `dotnet new` template
```
Comment (jmayntzhusen) [inline UmbracoNetCoreUpdates.md:82]:
```suggestion
### Steps to install the Umbraco `dotnet new` template
```

File: UmbracoNetCoreUpdates.md [hunk 2]
```diff
@@ -63,26 +63,47 @@ Found a bug that isn't already reported? Please report it on the [GitHub tracker
...
 
-2. Install the new Umbraco dotnet template:
+1. Install the new Umbraco dotnet template:
+
+    ```none
+    dotnet new -i Umbraco.Templates::0.5.0-alpha002
+    ```
+    
+### Steps to update the template from earlier alpha versions
+if you already has installed the Umbraco `dotnet new` template, you will need ensure it is up-to-date
```
Comment (jmayntzhusen) [inline UmbracoNetCoreUpdates.md:97]:
```suggestion
If you have already installed the Umbraco `dotnet new` template, you will need ensure it is up-to-date 
```

File: UmbracoNetCoreUpdates.md [hunk 3]
```diff
@@ -113,8 +134,8 @@ The following steps, will continue using CLI based on the steps above.
     dotnet run
     ```
 
-Even though you will not be able to see it from the CLI, the project is now running on the [Kestrel server](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/?view=aspnetcore-3.1&tabs=windows#kestrel) and be available on the default ports: http://localhost:5000 and https://localhost:5001.
+The project is now running on the [Kestrel server](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/?view=aspnetcore-3.1&tabs=windows#kestrel) and be available on the default ports: http://localhost:5000 and https://localhost:5001.
```
Comment (jmayntzhusen) [inline UmbracoNetCoreUpdates.md:137]:
```suggestion
The project is now running on the [Kestrel server](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/?view=aspnetcore-3.1&tabs=windows#kestrel) and is available on the default ports: http://localhost:5000 and https://localhost:5001.
```

**Review by jmayntzhusen — COMMENTED** [review]
Looks good Bjarke 🙂

Let me know when you want it merged in, found a few small typos here and there that I suggested some changes to.

File: UmbracoNetCoreUpdates.md [hunk 4]
```diff
@@ -63,26 +63,47 @@ Found a bug that isn't already reported? Please report it on the [GitHub tracker
...
+    ```
+    
+### Steps to update the template from earlier alpha versions
+If you have already installed the Umbraco `dotnet new` template, you will need ensure it is up-to-date 
+
+1. Use a command prompt of your choice to update the `dotnet new` templates
 
     ```none
-    dotnet new -i Umbraco.Templates::0.5.0-alpha001
+    dotnet new --update-apply
```
Comment (bergmania) [inline UmbracoNetCoreUpdates.md:102]:
```suggestion
    dotnet new -i Umbraco.Templates::0.5.0-alpha002
```

Apparrently the update-apply do not support prereleases

File: UmbracoNetCoreUpdates.md [hunk 5]
```diff
@@ -63,26 +63,47 @@ Found a bug that isn't already reported? Please report it on the [GitHub tracker
...
+### Known issues in current Alpha release
+
+ * Restarts during install
+    * When the Umbraco solution is installed, a restart is required. Right now we need to use IIS/IIS express to handle the next request and start the process again. Sometimes this fails and you need to start the process again
+ * Previews do not work due to the missing website part
+ * Members can’t be created, as we’re still using MembershipProviders
+ * Case sensitive filesystems (Linux and sometimes Mac) creates folders in the project like “~” and “**”, and lots of paths are not requested in the correct casing, yet
+ * Mac/Linux + Examine/Lucene issue as that assembly still is built for .NET Framework.
+ * External logins are not supported yet
+ * No support for Packages currently
```
Comment (bergmania) [inline UmbracoNetCoreUpdates.md:75]:
```suggestion
 * No support for Packages currently
 * Errors in log "Index Population failed for populator ..."
```

**Pull request merged**